src/components/AlbumItem.jsx

```diff
@@ -0,0 +1,59 @@
+import styles from '../styles/albumsList'
+
+import React, { Component } from 'react'
+// import classNames from 'classnames'
+import { Link, withRouter } from 'react-router'
+import { translate } from '../lib/I18n'
+
+import { getPhotoLink } from '../actions/photos'
+
+export class AlbumItem extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      isLoading: true
+    }
+
+    if (props.album && props.album.photosIds.length) {
+      getPhotoLink(props.album.photosIds[0])
+        .then(link => this.setState({
+          url: link,
+          isLoading: false
+        }))
+    } else {
+      this.setState({
+        url: '',
+        isLoading: false
+      })
+    }
+  }
+
+  render () {
+    const { t, album, router } = this.props
+    const { isLoading, url } = this.state
+    const parentPath = router.location.pathname
+    return (
+      !isLoading &&
+        <div className={styles['pho-album']}>
+          <Link
+            to={`${parentPath}/${album._id}`}
+            className={styles['pho-album-link']}
+          >
+            <img
+              className={styles['pho-album-photo-item']}
+              src={url}
```

Comment:
Not sure what we're doing elsewhere, but should we / can we set `alt` attributes for images?